model/knn/search_model/operations.smithy

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/search-plugins/knn/api/#search-model"
+)
+
+@xOperationGroup("knn.search_model")
```

Comment:
This should be pluralized as `knn.search_models`

Reply:
@Xtansia, made the requested changes.